mozilla-services/consvc-shepherd
# Add github action to build and deploy to personal dev environments

## References
no ticket

## Problem Statement

dev env needs a way to receive shepherd builds

## Proposed Changes

this pr adds a github action workflow file that calls the dev env's resuable workflow on git push to build/tag/deliver the docker image to dev env's container registry

[additional documentation](https://mozilla-hub.atlassian.net/wiki/spaces/MA1/pages/1056440462/poc+dev+env)

## Verification Steps

a github action run should start for the merge commit https://github.com/mozilla-services/consvc-shepherd/actions/workflows/dev_shepherd.yaml

## Visuals

no visuals
